tests/mechanic/launcher_test.py

```diff
@@ -174,12 +174,12 @@ def test_daemon_start_stop(self, wait_for_pidfile, chdir, get_size, supports, ja
         ms = get_metrics_store(cfg)
         proc_launcher = launcher.ProcessLauncher(cfg)
 
-        node_config = NodeConfiguration(build_type="tar", car_env={}, car_runtime_jdks="12,11", ip="127.0.0.1",
-                                        node_name="testnode", node_root_path="/tmp", binary_path="/tmp",
-                                        data_paths="/tmp")
+        node_configs = [NodeConfiguration(build_type="tar", car_env={}, car_runtime_jdks="12,11", ip="127.0.0.1",
+                                          node_name="testnode-{}".format(n), node_root_path="/tmp", binary_path="/tmp",
+                                          data_paths="/tmp") for n in range(2)]
```

Comment:
I think it would improve readability if we used a regular for-loop here instead of a list comprehension.

Reply:
Thanks! I changed it in https://github.com/elastic/rally/pull/883/commits/51c0df6046f26167f4664b71f76b85832ae70615.